Week1/homework/app.js

```diff
@@ -1,11 +1,143 @@
 'use strict';
-
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+  //the_brothers_karamazov
+  //three_men_in_a_boat
+  //ulysses
+  //the_great_gatsby
+  //war_and_peace
+  //the_odyssey
+  //crime_and_punishment
+  //the_stranger
+  //the_sun_also_rises
+  //candide
+
+  // 1.1
+  // const bookList = [
```

Comment:
You should keep using this array in the other functions.

Reply:
I have used this array in this function, do you want to mean something else?
//1.3
  // function general() {
  //   const newList = document.createElement('ul');

  //   for (let i = 0; i < bookList.length; i++) {
  //     const book = bookList[i];
  //     newList.innerHTML += `<li>${book}</li>`;
  //   }

  //   return newList;
  // }

  // document.body.append(general());